index.js

```diff
@@ -71,15 +71,17 @@ if (program.obj) {
  * searched first.
  */
 function parseObj (input) {
-  var str, out;
+  var str;
   try {
-    str = fs.readFileSync(program.obj);
+    str = fs.readFileSync(program.obj, 'utf8');
   } catch (e) {
-    return eval('(' + program.obj + ')');
+    str = program.obj;
+  }
+  try {
+    return JSON.parse(str);
+  } catch (e) {
+    return eval('(' + str + ')');
```

Comment:
why did we want parseObj to throw .parse exceptions in the past but now we catch them?